### PR TITLE
Splitted apply_model_schema_on_table method

### PR DESCRIPTION
**Pull request by AlexTorx**
Splitted apply_model_schema_on_table method into three more consistent parts regarding the different steps of migrating, loading slqlachemy events and reloading.

Originally, the apply_model_schema_on_table gathered either migrations, sqlalchemy ORm events registration and reload check. The execution of the whole method was conditionned by the registry.loadwithoutmigration attribute although sqlalchemy events registration whould not depend on whether system should perform a migration or not but must instead be executed at any time.

This PR is intented at proposing a better separation between migration and sqlalchemy orm events registration.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -69,6 +69,8 @@ install:
   - pip install -U pip
   - pip install git+https://github.com/jssuzanne/sqlalchemy-utils.git@mssql_and_pymssql#egg=sqlalchemy_utils
   - pip install wheel
+  - if [[ $TRAVIS_PYTHON_VERSION == 'pypy3.5' ]]; then pip install psycopg2cffi; else pip install psycopg2-binary; fi
+  - if [[ $SQLPYCLIENT == 'pymssql' ]]; then pip install 'pymssql<3.0'; fi
```
Comment (jssuzanne) [inline .travis.yml]: why dont modify the $SQLPYCLIENT ?

Comment (AlexTorx) [reply .travis.yml]: Actually yes it works ! ;) Corrected travis config file to only modify SQLPYCLIENT variable when needed.

**Comment by jssuzanne** [comment]
Great, a last miss, can you change the changelog

**Comment by AlexTorx** [comment]
> Great, a last miss, can you change the changelog

Yep, done !

**Comment by jssuzanne** [comment]
Thank for the report and the fix

**Pull request merged**